### PR TITLE
ACMS-1468: Drupal 10 Readiness - Update contrib version in ACMS Site Studio, Image, Search Module.

### PR DESCRIPTION
**Pull request by rajeshreeputra**
**Motivation**
Fixes #ACMS-1468

**Proposed changes**
Drupal 10 Readiness - Update contrib version in ACMS Image, Search, Site Studio Module.

**Alternatives considered**
NA

**Testing steps**

- Install site with this code `/vendor/bin/acms site:install`.
- Verify everything works fine.

**Merge requirements**
- [ ] _Major change_, _Minor change_, _Bug_, _Enhancement_, and/or _Chore_ label applied
- [ ] Manual testing by a reviewer

### CODE REVIEW

**Review by deepakmishra2 — APPROVED** [review]
Looks good to me

File: modules/acquia_cms_image/composer.json [hunk 1]
```diff
@@ -22,7 +22,11 @@
     },
     "extra": {
         "enable-patching": true,
-        "patches": {}
+        "patches": {
+            "drupal/imce": {
+                "3324281 - Property label with value Imce Link does not exist CKEditor5PluginDefinition": "https://www.drupal.org/files/issues/2022-11-30/3324281-7.patch"
+            }
+        }
```
Comment (deepakmishra2) [inline modules/acquia_cms_image/composer.json:29]: I think this patch is already getting applied in this PR - https://github.com/acquia/acquia_cms/pull/1331/files

File: modules/acquia_cms_site_studio/composer.json [hunk 2]
```diff
@@ -4,11 +4,11 @@
     "description": "Handles code for Site Studio Installation & Related Configurations.",
     "license": "GPL-2.0-or-later",
     "require": {
-        "acquia/cohesion": "~6.9.0",
-        "acquia/cohesion-theme": "^6.8",
-        "drupal/acquia_cms_common": "^1.5",
-        "drupal/collapsiblock": "^3",
-        "drupal/node_revision_delete": "^1.0@RC"
+        "acquia/cohesion": "~7.0",
+        "acquia/cohesion-theme": "~7.0",
```
Comment (chandan-singh7929) [inline modules/acquia_cms_site_studio/composer.json:8]: Please use `^` instead of `~`

Comment (rajeshreeputra) [reply modules/acquia_cms_site_studio/composer.json]: done

File: modules/acquia_cms_search/src/Facade/AcquiaSearchFacade.php [hunk 3]
```diff
@@ -102,12 +101,14 @@ public static function submitSettingsForm() : void {
    *   FALSE.
    */
   private static function isConfigured() : bool {
-    return (
-      (bool) AcquiaSearch::getIdentifier() &&
-      (bool) AcquiaSearch::getApiKey() &&
-      (bool) AcquiaSearch::getApiHost() &&
-      (bool) AcquiaSearch::getUuid()
-    );
+    $api_host = \Drupal::configFactory()->getEditable('acquia_search.settings')->get('api_host');
```
[the remark this answers is not in the document]
Comment (rajeshreeputra) [reply modules/acquia_cms_search/src/Facade/AcquiaSearchFacade.php:104]: @deepakmishra2 - please use DI from PR-1343

[the remark this answers is not in the document]
Comment (deepakmishra2) [reply modules/acquia_cms_search/src/Facade/AcquiaSearchFacade.php]: @chandan-singh7929 @rajeshreeputra added changes from PR-1343

**Pull request merged**